zuul.d/base.yaml

```diff
@@ -114,7 +114,7 @@
         - name: compute-0
           label: cloud-centos-9-stream-tripleo-vexxhost
         - name: crc
-          label: coreos-crc-extracted-xxl
+          label: coreos-crc-extracted-3xl
```

Comment:
I would change this only in `podified-multinode-edpm-ironic-nobuild-tagged-crc` job, since here will affect lot of jobs and increase resource consumption.

Reply:
done